alibi_detect/cd/base_online.py

```diff
@@ -99,7 +125,7 @@ def reset(self) -> None:
         "Resets the detector but does not reconfigure thresholds."
         self._initialise()
 
-    def predict(self, x_t: Union[np.ndarray, Any],  return_test_stat: bool = True,
+    def predict(self, x_t: Union[np.ndarray, list],  return_test_stat: bool = True,
```

Comment:
I don't think that changing form `Any` to `list` as type for `x_t` is correct? We're passing single instances to `predict`, which can be e.g. arrays, string values or custom objects. They are turned into batches (array or list) of size 1 by e.g. `x_t[None, :]` or `[x_t]`.

Reply:
Thanks, good point. I shall change this back now!